src/test/scripts/helm_install.sh

```diff
@@ -6,11 +6,17 @@ set -x
 getCurrentClusterType() {
```

Comment:
TIL Bash is even worse than I used to think. No return values and returning by echoing 🤦 .

Reply:
well, you can return a status. You can also return via a param outpu variable, I don't particularly like that syntax, though.